RucioClient.py

```diff
@@ -91,8 +91,10 @@ def getFileCountPerBlock(self, dataset):
             blocks = []
             for block in self.getBlockNamesDataset(dataset):
                 blocks.append((block, self.getFileCountBlock(block)))
+            if blocks is None:
```

Comment:
blocks=[] with subsequent append cannot turn into None, ever. 
are you actually observing this ?

Reply:
If you see the GH issue, checkor reports issue on L1041 where the None is coming from ‘all_filecount_pb’ and that is the sum of ‘getfilecountperblock’ for Phedex and rucio! I don’t see how changing ‘getfilecountblock’ would help here as that function isn’t being called!

Reply:
@vlimant for fixing `getFileCountBlock()`.  I simply did not expect to see a `None` value returned by Rucio here. BTW @sharad1126 this function is called and it is exactly here: 
https://github.com/CMSCompOps/WmAgentScripts/blob/1eb640459848d65de6188d44e5a42fe789098091/RucioClient.py#L93
And this is exactly how the `None` value enters the final block list. This `None` is not returned as  the full block list but it rather enters as one of it's elements.

And this return a `0` instead of `[]` in `getFileCountPerBlock()`, somehow slipped between the lines. Thanks for catching that  one too.